<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="dcterms.date" content="2025-02-17">

<title>Neuroscience methods – PSY 511.003</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-7931e10a13f2f51d5932bd5e1fda4e06.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script type="application/json" class="js-hypothesis-config">
{
  "theme": "clean"
}
</script>
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="../include/css/styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">PSY 511.003</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../schedule.html"> 
<span class="menu-text">Schedule</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../evaluation.html"> 
<span class="menu-text">Evaluation</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../policies.html"> 
<span class="menu-text">Policies</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-exercises" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Exercises</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-exercises">    
        <li>
    <a class="dropdown-item" href="../exercises/ex01.pdf">
 <span class="dropdown-text">Exercise 01 • Levels/Terms</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/ex02.pdf">
 <span class="dropdown-text">Exercise 02 • Neuroanatomy</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/ex03.pdf">
 <span class="dropdown-text">Exercise 03 • Neurophysiology I</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/final-project.pdf">
 <span class="dropdown-text">Final project</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-resources" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Resources</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-resources">    
        <li>
    <a class="dropdown-item" href="../resources/anatomy.html">
 <span class="dropdown-text">Neuroanatomy</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../resources/cells.html">
 <span class="dropdown-text">Cellular neuroscience</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../resources/neurochemistry.html">
 <span class="dropdown-text">Neurochemistry</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-supplemental" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Supplemental</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-supplemental">    
        <li>
    <a class="dropdown-item" href="../supplemental/ex01-themes.html">
 <span class="dropdown-text">Exercise 01 Themes</span></a>
  </li>  
    </ul>
  </li>
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-reader-toggle quarto-navigation-tool px-1" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#evaluating-methods" id="toc-evaluating-methods" class="nav-link active" data-scroll-target="#evaluating-methods">Evaluating methods</a>
  <ul>
  <li><a href="#what-are-we-measuring" id="toc-what-are-we-measuring" class="nav-link" data-scroll-target="#what-are-we-measuring">What are we measuring?</a></li>
  <li><a href="#what-is-the-question" id="toc-what-is-the-question" class="nav-link" data-scroll-target="#what-is-the-question">What is the question?</a></li>
  <li><a href="#strengths-weaknesses" id="toc-strengths-weaknesses" class="nav-link" data-scroll-target="#strengths-weaknesses">Strengths &amp; Weaknesses</a></li>
  <li><a href="#spatial-resolution" id="toc-spatial-resolution" class="nav-link" data-scroll-target="#spatial-resolution">Spatial resolution</a></li>
  <li><a href="#and-temporal-resolution" id="toc-and-temporal-resolution" class="nav-link" data-scroll-target="#and-temporal-resolution">…and temporal resolution</a></li>
  <li><a href="#types-of-methods" id="toc-types-of-methods" class="nav-link" data-scroll-target="#types-of-methods">Types of methods</a></li>
  </ul></li>
  <li><a href="#structural-methods" id="toc-structural-methods" class="nav-link" data-scroll-target="#structural-methods">Structural methods</a>
  <ul>
  <li><a href="#mapping-microstructure" id="toc-mapping-microstructure" class="nav-link" data-scroll-target="#mapping-microstructure">Mapping microstructure</a>
  <ul class="collapse">
  <li><a href="#golgi-stain" id="toc-golgi-stain" class="nav-link" data-scroll-target="#golgi-stain">Golgi stain</a></li>
  <li><a href="#camillo-golgi" id="toc-camillo-golgi" class="nav-link" data-scroll-target="#camillo-golgi">Camillo Golgi</a></li>
  <li><a href="#nissl-stain" id="toc-nissl-stain" class="nav-link" data-scroll-target="#nissl-stain">Nissl stain</a></li>
  <li><a href="#franz-nissl" id="toc-franz-nissl" class="nav-link" data-scroll-target="#franz-nissl">Franz Nissl</a></li>
  <li><a href="#brainbow" id="toc-brainbow" class="nav-link" data-scroll-target="#brainbow">Brainbow</a></li>
  <li><a href="#clarity" id="toc-clarity" class="nav-link" data-scroll-target="#clarity">Clarity</a></li>
  <li><a href="#cubic" id="toc-cubic" class="nav-link" data-scroll-target="#cubic">CUBIC</a></li>
  <li><a href="#evaluating-microcellular-techniques" id="toc-evaluating-microcellular-techniques" class="nav-link" data-scroll-target="#evaluating-microcellular-techniques">Evaluating micro/cellular techniques</a></li>
  </ul></li>
  <li><a href="#mapping-macro-structures" id="toc-mapping-macro-structures" class="nav-link" data-scroll-target="#mapping-macro-structures">Mapping macro-structures</a>
  <ul class="collapse">
  <li><a href="#computed-axial-tomography-cat-ct" id="toc-computed-axial-tomography-cat-ct" class="nav-link" data-scroll-target="#computed-axial-tomography-cat-ct">Computed axial tomography (CAT), CT</a></li>
  <li><a href="#magnetic-resonance-imaging-mri" id="toc-magnetic-resonance-imaging-mri" class="nav-link" data-scroll-target="#magnetic-resonance-imaging-mri">Magnetic Resonance Imaging (MRI)</a></li>
  </ul></li>
  <li><a href="#mapping-the-wiring-diagram-connectome" id="toc-mapping-the-wiring-diagram-connectome" class="nav-link" data-scroll-target="#mapping-the-wiring-diagram-connectome">Mapping the wiring diagram (“connectome”)</a>
  <ul class="collapse">
  <li><a href="#microscale" id="toc-microscale" class="nav-link" data-scroll-target="#microscale">Microscale</a></li>
  <li><a href="#diffusion-tensor-imaging-dti" id="toc-diffusion-tensor-imaging-dti" class="nav-link" data-scroll-target="#diffusion-tensor-imaging-dti">Diffusion Tensor Imaging (DTI)</a></li>
  <li><a href="#visualizing-the-connectome" id="toc-visualizing-the-connectome" class="nav-link" data-scroll-target="#visualizing-the-connectome">Visualizing the connectome</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#functional-methods" id="toc-functional-methods" class="nav-link" data-scroll-target="#functional-methods">Functional methods</a>
  <ul>
  <li><a href="#recording-from-the-brain" id="toc-recording-from-the-brain" class="nav-link" data-scroll-target="#recording-from-the-brain">Recording from the brain</a>
  <ul class="collapse">
  <li><a href="#singlemulti-unit-recording" id="toc-singlemulti-unit-recording" class="nav-link" data-scroll-target="#singlemulti-unit-recording">Single/multi-unit Recording</a></li>
  <li><a href="#electrocorticography-ecog" id="toc-electrocorticography-ecog" class="nav-link" data-scroll-target="#electrocorticography-ecog">Electrocorticography (ECoG)</a></li>
  <li><a href="#positron-emission-tomography-pet" id="toc-positron-emission-tomography-pet" class="nav-link" data-scroll-target="#positron-emission-tomography-pet">Positron Emission Tomography (PET)</a></li>
  <li><a href="#functional-magnetic-resonance-imaging-fmri" id="toc-functional-magnetic-resonance-imaging-fmri" class="nav-link" data-scroll-target="#functional-magnetic-resonance-imaging-fmri">Functional Magnetic Resonance Imaging (fMRI)</a></li>
  <li><a href="#functional-near-infrared-spectroscopy-fnirs" id="toc-functional-near-infrared-spectroscopy-fnirs" class="nav-link" data-scroll-target="#functional-near-infrared-spectroscopy-fnirs">Functional Near-infrared Spectroscopy (fNIRS)</a></li>
  <li><a href="#electroencephalography-eeg" id="toc-electroencephalography-eeg" class="nav-link" data-scroll-target="#electroencephalography-eeg">Electroencephalography (EEG)</a></li>
  <li><a href="#magneto-encephalography-meg" id="toc-magneto-encephalography-meg" class="nav-link" data-scroll-target="#magneto-encephalography-meg">Magneto-encephalography (MEG)</a></li>
  <li><a href="#how-do-eegmeg-and-fmri-relate" id="toc-how-do-eegmeg-and-fmri-relate" class="nav-link" data-scroll-target="#how-do-eegmeg-and-fmri-relate">How do EEG/MEG and fMRI relate?</a></li>
  </ul></li>
  <li><a href="#manipulating-the-brain" id="toc-manipulating-the-brain" class="nav-link" data-scroll-target="#manipulating-the-brain">Manipulating the brain</a>
  <ul class="collapse">
  <li><a href="#interfering-with-the-brain" id="toc-interfering-with-the-brain" class="nav-link" data-scroll-target="#interfering-with-the-brain">Interfering with the brain</a></li>
  <li><a href="#stimulating-the-brain" id="toc-stimulating-the-brain" class="nav-link" data-scroll-target="#stimulating-the-brain">Stimulating the brain</a></li>
  <li><a href="#optogenetics" id="toc-optogenetics" class="nav-link" data-scroll-target="#optogenetics">Optogenetics</a></li>
  </ul></li>
  <li><a href="#simulating-the-brain" id="toc-simulating-the-brain" class="nav-link" data-scroll-target="#simulating-the-brain"><em>Sim</em>ulating the brain</a></li>
  <li><a href="#or-synthesizing-one" id="toc-or-synthesizing-one" class="nav-link" data-scroll-target="#or-synthesizing-one">Or synthesizing one…</a>
  <ul class="collapse">
  <li><a href="#human-brain-organoids" id="toc-human-brain-organoids" class="nav-link" data-scroll-target="#human-brain-organoids">Human brain organoids</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/psu-psychology/psy-511-scan-fdns-2025-spring/edit/main/src/resources/methods.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/psu-psychology/psy-511-scan-fdns-2025-spring/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Neuroscience methods</h1>
</div>



<div class="quarto-title-meta">

    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 17, 2025</p>
    </div>
  </div>
  
    <div>
    <div class="quarto-title-meta-heading">Modified</div>
    <div class="quarto-title-meta-contents">
      <p class="date-modified">February 17, 2025</p>
    </div>
  </div>
    
  </div>
  


</header>


<section id="evaluating-methods" class="level2">
<h2 class="anchored" data-anchor-id="evaluating-methods">Evaluating methods</h2>
<section id="what-are-we-measuring" class="level3">
<h3 class="anchored" data-anchor-id="what-are-we-measuring">What are we measuring?</h3>
<ul>
<li>Structure</li>
<li>Activity
<ul>
<li>Why not <em>function</em>?</li>
</ul></li>
</ul>
</section>
<section id="what-is-the-question" class="level3">
<h3 class="anchored" data-anchor-id="what-is-the-question">What is the question?</h3>
<ul>
<li>Structure X -&gt; Structure Y</li>
<li>Structure X -&gt; Function Y</li>
</ul>
</section>
<section id="strengths-weaknesses" class="level3">
<h3 class="anchored" data-anchor-id="strengths-weaknesses">Strengths &amp; Weaknesses</h3>
<ul>
<li>Cost</li>
<li>Invasiveness</li>
<li>Spatial/temporal resolution</li>
</ul>
</section>
<section id="spatial-resolution" class="level3">
<h3 class="anchored" data-anchor-id="spatial-resolution">Spatial resolution</h3>
<p><img src="../include/img/churchland-levels-of-analysis.gif" class="img-fluid"></p>
</section>
<section id="and-temporal-resolution" class="level3">
<h3 class="anchored" data-anchor-id="and-temporal-resolution">…and temporal resolution</h3>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.3839/MediaObjects/41593_2014_Article_BFnn3839_Fig1_HTML.jpg?as=webp" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="@Sejnowski2014-aa"><img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.3839/MediaObjects/41593_2014_Article_BFnn3839_Fig1_HTML.jpg?as=webp" class="img-fluid figure-img" alt="Sejnowski, Churchland, &amp; Movshon (2014)"></a></p>
<figcaption><span class="citation" data-cites="Sejnowski2014-aa">Sejnowski et al. (<a href="#ref-Sejnowski2014-aa" role="doc-biblioref">2014</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="types-of-methods" class="level3">
<h3 class="anchored" data-anchor-id="types-of-methods">Types of methods</h3>
<ul>
<li>Structural
<ul>
<li>Anatomy</li>
<li>Connectivity/connectome</li>
</ul></li>
<li>Functional
<ul>
<li>What does it do?</li>
<li>Physiology/Activity</li>
</ul></li>
</ul>
</section>
</section>
<section id="structural-methods" class="level2">
<h2 class="anchored" data-anchor-id="structural-methods">Structural methods</h2>
<section id="mapping-microstructure" class="level3">
<h3 class="anchored" data-anchor-id="mapping-microstructure">Mapping microstructure</h3>
<ul>
<li>Cell/axon stains</li>
<li>Cellular distribution, concentration, microanatomy</li>
</ul>
<section id="golgi-stain" class="level4">
<h4 class="anchored" data-anchor-id="golgi-stain">Golgi stain</h4>
<ul>
<li>whole cells, but small % of total</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/golgi-stain.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg"><img src="../include/img/golgi-stain.jpg" class="img-fluid figure-img" alt="http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg"></a></p>
<figcaption><a href="http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg" class="uri">http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg</a></figcaption>
</figure>
</div>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://the-public-domain-review.imgix.net/collections/illustrations-of-the-nervous-system-golgi-and-cajal/golgi-olfactory-top-thumb.jpg?fit=max&amp;w=1200&amp;h=850" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="https://publicdomainreview.org/collection/illustrations-of-the-nervous-system-golgi-and-cajal/"><img src="https://the-public-domain-review.imgix.net/collections/illustrations-of-the-nervous-system-golgi-and-cajal/golgi-olfactory-top-thumb.jpg?fit=max&amp;w=1200&amp;h=850" class="img-fluid figure-img" alt="https://publicdomainreview.org/collection/illustrations-of-the-nervous-system-golgi-and-cajal/"></a></p>
<figcaption><a href="https://publicdomainreview.org/collection/illustrations-of-the-nervous-system-golgi-and-cajal/" class="uri">https://publicdomainreview.org/collection/illustrations-of-the-nervous-system-golgi-and-cajal/</a></figcaption>
</figure>
</div>
</section>
<section id="camillo-golgi" class="level4">
<h4 class="anchored" data-anchor-id="camillo-golgi"><a href="https://en.wikipedia.org/wiki/Camillo_Golgi">Camillo Golgi</a></h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Camillo_Golgi.jpg/440px-Camillo_Golgi.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Camillo Golgi (Wikipedia)"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Camillo_Golgi.jpg/440px-Camillo_Golgi.jpg" class="img-fluid figure-img" alt="Camillo Golgi (Wikipedia)"></a></p>
<figcaption>Camillo Golgi (Wikipedia)</figcaption>
</figure>
</div>
</section>
<section id="nissl-stain" class="level4">
<h4 class="anchored" data-anchor-id="nissl-stain">Nissl stain</h4>
<ul>
<li>Only cell bodies</li>
<li>Cell density ~ color intensity</li>
</ul>
<p><a href="https://i.pinimg.com/originals/24/ba/58/24ba5870a0b3ac2ce8e3620853e12c8b.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-5"><img src="https://i.pinimg.com/originals/24/ba/58/24ba5870a0b3ac2ce8e3620853e12c8b.jpg" class="img-fluid"></a></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.researchgate.net/publication/355182651/figure/fig4/AS:1077963713970180@1634017985463/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="https://www.researchgate.net/figure/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the_fig4_355182651"><img src="https://www.researchgate.net/publication/355182651/figure/fig4/AS:1077963713970180@1634017985463/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the.png" class="img-fluid figure-img" alt="https://www.researchgate.net/figure/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the_fig4_355182651"></a></p>
<figcaption><a href="https://www.researchgate.net/figure/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the_fig4_355182651" class="uri">https://www.researchgate.net/figure/Nissl-staining-and-Golgi-stain-in-rats-30days-after-infarction-A-Nissl-staining-of-the_fig4_355182651</a></figcaption>
</figure>
</div>
</section>
<section id="franz-nissl" class="level4">
<h4 class="anchored" data-anchor-id="franz-nissl">Franz Nissl</h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Franz_Nissl.jpg/440px-Franz_Nissl.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-7" title="Franz Nissl (Wikipedia)"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Franz_Nissl.jpg/440px-Franz_Nissl.jpg" class="img-fluid figure-img" alt="Franz Nissl (Wikipedia)"></a></p>
<figcaption>Franz Nissl (Wikipedia)</figcaption>
</figure>
</div>
</section>
<section id="brainbow" class="level4">
<h4 class="anchored" data-anchor-id="brainbow"><a href="http://cbs.fas.harvard.edu/science/connectome-project/brainbow">Brainbow</a></h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/lichtman-nrn2391-f1.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-8" title="@lichtman_technicolour_2008"><img src="../include/img/lichtman-nrn2391-f1.jpg" class="img-fluid figure-img" alt="Lichtman, Livet, &amp; Sanes (2008)"></a></p>
<figcaption><span class="citation" data-cites="lichtman_technicolour_2008">Lichtman et al. (<a href="#ref-lichtman_technicolour_2008" role="doc-biblioref">2008</a>)</span></figcaption>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/lichtman-nrn2391-f2.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-9" title="@lichtman_technicolour_2008"><img src="../include/img/lichtman-nrn2391-f2.jpg" class="img-fluid figure-img" alt="Lichtman et al. (2008)"></a></p>
<figcaption><span class="citation" data-cites="lichtman_technicolour_2008">Lichtman et al. (<a href="#ref-lichtman_technicolour_2008" role="doc-biblioref">2008</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="clarity" class="level4">
<h4 class="anchored" data-anchor-id="clarity"><a href="http://clarityresourcecenter.com/CLARITY.html">Clarity</a></h4>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/c-NMfp13Uug" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/embed/c-NMfp13Uug" class="uri">https://www.youtube.com/embed/c-NMfp13Uug</a></p>
</section>
<section id="cubic" class="level4">
<h4 class="anchored" data-anchor-id="cubic"><a href="https://en.wikipedia.org/wiki/CUBIC">CUBIC</a></h4>
<ul>
<li>CUBIC (“clear, unobstructed brain/body imaging cocktails and computational analysis”)</li>
<li><span class="citation" data-cites="Susaki2014-xj">(<a href="#ref-Susaki2014-xj" role="doc-biblioref">Susaki et al., 2014</a>)</span></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://ars.els-cdn.com/content/image/1-s2.0-B9780444636393000219-f21-02-9780444636393.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-10" title="Fig 21.2 from @Ando2018-ze. Immunostaining of CUBIC-clarified 500-μm-thick slices from human Alzheimer disease postmortem brain frontal cortex. Human Alzheimer disease frontal cortex tissue immunostained for Aß (6E10, red) and for tau (B19, green). Stack depth of 264 μm; step size = 1 μm. Stack photos were taken with a two-photon microscope equipped with a 20 × air objective. Scale bar, 100 μm."><img src="https://ars.els-cdn.com/content/image/1-s2.0-B9780444636393000219-f21-02-9780444636393.jpg" class="img-fluid figure-img" alt="Fig 21.2 from Ando, Laborde, Brion, &amp; Duyckaerts (2018). Immunostaining of CUBIC-clarified 500-μm-thick slices from human Alzheimer disease postmortem brain frontal cortex. Human Alzheimer disease frontal cortex tissue immunostained for Aß (6E10, red) and for tau (B19, green). Stack depth of 264 μm; step size = 1 μm. Stack photos were taken with a two-photon microscope equipped with a 20 × air objective. Scale bar, 100 μm."></a></p>
<figcaption>Fig 21.2 from <span class="citation" data-cites="Ando2018-ze">Ando et al. (<a href="#ref-Ando2018-ze" role="doc-biblioref">2018</a>)</span>. Immunostaining of CUBIC-clarified 500-μm-thick slices from human Alzheimer disease postmortem brain frontal cortex. Human Alzheimer disease frontal cortex tissue immunostained for Aß (6E10, red) and for tau (B19, green). Stack depth of 264 μm; step size = 1 μm. Stack photos were taken with a two-photon microscope equipped with a 20 × air objective. Scale bar, 100 μm.</figcaption>
</figure>
</div>
</section>
<section id="evaluating-microcellular-techniques" class="level4">
<h4 class="anchored" data-anchor-id="evaluating-microcellular-techniques">Evaluating micro/cellular techniques</h4>
<ul>
<li>Invasive (in humans post-mortem only)</li>
<li>High <em>spatial</em> resolution, but poor/coarse <em>temporal</em></li>
</ul>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>“SHANK2 mutations associated with autism spectrum disorder cause hyperconnectivity of human neurons.” <span class="citation" data-cites="Zaslavsky2019-ln">(<a href="#ref-Zaslavsky2019-ln" role="doc-biblioref">Zaslavsky et al., 2019</a>)</span></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-019-0365-8/MediaObjects/41593_2019_365_Fig1_HTML.png?as=webp" class="lightbox" data-gallery="quarto-lightbox-gallery-11" title="Figure 1 from @Zaslavsky2019-ln. a, iPSCs generated from multiple control and affected individuals are differentiated into NPCs. NPCs are differentiated in separate wells for 4 weeks and then differentially fluorescently labeled control (CTRL) and mutant (MUT) cells are sparsely seeded onto a large unlabeled neuronal population (the lawn) and cocultured with astrocytes. b, Timeline of the experiment, starting with seeding of NPCs. Measurements of mutant cells are normalized to control cells in the same well. c, Sparse seeding allows simultaneous analyses of cell morphology and connectivity (total number of SYN1 puncta) of single neurons. Scale bars, 100 μm. d, To compare cell morphology, paired representative traces are shown of control and SHANK2 ASD or engineered SHANK2 KO neurons grown in the same well. e, To compare synaptic function, sEPSCs are recorded from neurons grown in the same well. Confocal images and traces shown in c and d are representative of iPSC-derived neurons imaged in experiments depicted in Fig. 2a–c.&nbsp;sEPSC traces shown in e are representative of patch-clamp recordings of iPSC-derived neurons described in Fig. 3."><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-019-0365-8/MediaObjects/41593_2019_365_Fig1_HTML.png?as=webp" class="img-fluid figure-img" alt="Figure 1 from Zaslavsky et al. (2019). a, iPSCs generated from multiple control and affected individuals are differentiated into NPCs. NPCs are differentiated in separate wells for 4 weeks and then differentially fluorescently labeled control (CTRL) and mutant (MUT) cells are sparsely seeded onto a large unlabeled neuronal population (the lawn) and cocultured with astrocytes. b, Timeline of the experiment, starting with seeding of NPCs. Measurements of mutant cells are normalized to control cells in the same well. c, Sparse seeding allows simultaneous analyses of cell morphology and connectivity (total number of SYN1 puncta) of single neurons. Scale bars, 100 μm. d, To compare cell morphology, paired representative traces are shown of control and SHANK2 ASD or engineered SHANK2 KO neurons grown in the same well. e, To compare synaptic function, sEPSCs are recorded from neurons grown in the same well. Confocal images and traces shown in c and d are representative of iPSC-derived neurons imaged in experiments depicted in Fig. 2a–c.&nbsp;sEPSC traces shown in e are representative of patch-clamp recordings of iPSC-derived neurons described in Fig. 3."></a></p>
<figcaption>Figure 1 from <span class="citation" data-cites="Zaslavsky2019-ln">Zaslavsky et al. (<a href="#ref-Zaslavsky2019-ln" role="doc-biblioref">2019</a>)</span>. a, iPSCs generated from multiple control and affected individuals are differentiated into NPCs. NPCs are differentiated in separate wells for 4 weeks and then differentially fluorescently labeled control (CTRL) and mutant (MUT) cells are sparsely seeded onto a large unlabeled neuronal population (the lawn) and cocultured with astrocytes. b, Timeline of the experiment, starting with seeding of NPCs. Measurements of mutant cells are normalized to control cells in the same well. c, Sparse seeding allows simultaneous analyses of cell morphology and connectivity (total number of SYN1 puncta) of single neurons. Scale bars, 100 μm. d, To compare cell morphology, paired representative traces are shown of control and SHANK2 ASD or engineered SHANK2 KO neurons grown in the same well. e, To compare synaptic function, sEPSCs are recorded from neurons grown in the same well. Confocal images and traces shown in c and d are representative of iPSC-derived neurons imaged in experiments depicted in Fig. 2a–c.&nbsp;sEPSC traces shown in e are representative of patch-clamp recordings of iPSC-derived neurons described in Fig. 3.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="mapping-macro-structures" class="level3">
<h3 class="anchored" data-anchor-id="mapping-macro-structures">Mapping macro-structures</h3>
<section id="computed-axial-tomography-cat-ct" class="level4">
<h4 class="anchored" data-anchor-id="computed-axial-tomography-cat-ct">Computed axial tomography (CAT), CT</h4>
<ul>
<li>X-ray based</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/tomography.png" class="lightbox" data-gallery="quarto-lightbox-gallery-12" title="http://img.tfd.com/mk/T/X2604-T-22.png"><img src="../include/img/tomography.png" class="img-fluid figure-img" alt="http://img.tfd.com/mk/T/X2604-T-22.png"></a></p>
<figcaption>http://img.tfd.com/mk/T/X2604-T-22.png</figcaption>
</figure>
</div>
<section id="tomography" class="level5">
<h5 class="anchored" data-anchor-id="tomography">Tomography</h5>
<ul>
<li>Structured sequence of 2D images -&gt; 3D reconstruction</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/cat-scan-pineapple.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-13" title="http://static.howstuffworks.com/gif/cat-scan-pineapple.jpg"><img src="../include/img/cat-scan-pineapple.jpg" class="img-fluid figure-img" alt="http://static.howstuffworks.com/gif/cat-scan-pineapple.jpg"></a></p>
<figcaption>http://static.howstuffworks.com/gif/cat-scan-pineapple.jpg</figcaption>
</figure>
</div>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://miro.medium.com/max/1024/1*j09PgBpdJIHXhlyjcby0HA.jpeg" class="lightbox" data-gallery="quarto-lightbox-gallery-14" title="https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7"><img src="https://miro.medium.com/max/1024/1*j09PgBpdJIHXhlyjcby0HA.jpeg" class="img-fluid figure-img" alt="https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7"></a></p>
<figcaption><a href="https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7" class="uri">https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7</a></figcaption>
</figure>
</div>
</section>
</section>
<section id="magnetic-resonance-imaging-mri" class="level4">
<h4 class="anchored" data-anchor-id="magnetic-resonance-imaging-mri">Magnetic Resonance Imaging (MRI)</h4>
<ul>
<li>Magnetic resonance a property of some isotopes and complex molecules</li>
<li>Hydrogen (<span class="math inline">\(H\)</span>), common in water &amp; fat, is one</li>
<li>In magnetic field, <span class="math inline">\(H\)</span> atoms absorb and release radio frequency (RF) energy</li>
<li><span class="math inline">\(H\)</span> atoms align with strong magnetic field</li>
<li>Applying RF pulse perturbs alignment</li>
<li>Rate/timing of realignment varies by tissue</li>
<li>Realignment gives off radio frequency (RF) signals</li>
<li>Strength of RF ~ density of <span class="math inline">\(H\)</span> (or other target)</li>
<li><a href="https://radiopaedia.org/articles/k-space-1?lang=us"><span class="math inline">\(k\)</span>-space</a> (frequency/phase) -&gt; anatomical space
<ul>
<li><a href="https://en.wikipedia.org/wiki/Fourier_transform">Fourier analysis and synthesis</a> important for psychological/neural science, but possibly unfamiliar to many.</li>
<li>Frequency and phase <span class="math inline">\(/rightarrow\)</span> amplitude and location</li>
</ul></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/mri-steps.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-15" title="http://s.hswstatic.com/gif/mri-steps.jpg"><img src="../include/img/mri-steps.jpg" class="img-fluid figure-img" alt="http://s.hswstatic.com/gif/mri-steps.jpg"></a></p>
<figcaption>http://s.hswstatic.com/gif/mri-steps.jpg</figcaption>
</figure>
</div>
<section id="structural-mri" class="level5">
<h5 class="anchored" data-anchor-id="structural-mri">Structural MRI</h5>
<ul>
<li>Tissue density/type differences</li>
<li><strong>Gray matter</strong> (nerve cells &amp; <strong>dendrites</strong>) vs.&nbsp;<strong>white matter</strong> (<strong>axon fibers</strong>)</li>
</ul>
<p><a href="https://previews.123rf.com/images/ultimagaina/ultimagaina1405/ultimagaina140500005/28078449-Sequence-of-vertical-sections-of-a-human-brain-MRI-scan-Stock-Photo.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-16"><img src="https://previews.123rf.com/images/ultimagaina/ultimagaina1405/ultimagaina140500005/28078449-Sequence-of-vertical-sections-of-a-human-brain-MRI-scan-Stock-Photo.jpg" class="img-fluid"></a></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/a462a956-7f5d-4995-9989-647b2823c66f/gr1_lrg.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-17" title="Figure 1 from @Lee2021-ae. Longitudinal trajectories of total cerebral volume, gray matter volume, and white matter volume from early to middle childhood (A) in boys with autism spectrum disorder (ASD) and typically developing (TD) boys and (B) in boys with ASD and disproportionate megalencephaly (ASD-DM), boys with ASD with normative cerebral volume-to-height ratio (ASD-N), and TD boys."><img src="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/a462a956-7f5d-4995-9989-647b2823c66f/gr1_lrg.jpg" class="img-fluid figure-img" alt="Figure 1 from Lee et al. (2021). Longitudinal trajectories of total cerebral volume, gray matter volume, and white matter volume from early to middle childhood (A) in boys with autism spectrum disorder (ASD) and typically developing (TD) boys and (B) in boys with ASD and disproportionate megalencephaly (ASD-DM), boys with ASD with normative cerebral volume-to-height ratio (ASD-N), and TD boys."></a></p>
<figcaption>Figure 1 from <span class="citation" data-cites="Lee2021-ae">Lee et al. (<a href="#ref-Lee2021-ae" role="doc-biblioref">2021</a>)</span>. Longitudinal trajectories of total cerebral volume, gray matter volume, and white matter volume from early to middle childhood (A) in boys with autism spectrum disorder (ASD) and typically developing (TD) boys and (B) in boys with ASD and disproportionate megalencephaly (ASD-DM), boys with ASD with normative cerebral volume-to-height ratio (ASD-N), and TD boys.</figcaption>
</figure>
</div>
</section>
<section id="mr-spectroscopy-specific-metabolites" class="level5">
<h5 class="anchored" data-anchor-id="mr-spectroscopy-specific-metabolites">MR Spectroscopy (specific metabolites)</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.brighamandwomens.org/assets/BWH/radiology/images/spectroscopy-about-spectroscopy.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-18" title="https://www.brighamandwomens.org/radiology/research/spectroscopy/about-spectroscopy"><img src="https://www.brighamandwomens.org/assets/BWH/radiology/images/spectroscopy-about-spectroscopy.jpg" class="img-fluid figure-img" alt="https://www.brighamandwomens.org/radiology/research/spectroscopy/about-spectroscopy"></a></p>
<figcaption>https://www.brighamandwomens.org/radiology/research/spectroscopy/about-spectroscopy</figcaption>
</figure>
</div>
<ul>
<li>Region sizes/volumes</li>
</ul>
</section>
<section id="voxel-based-morphometry-vbm" class="level5">
<h5 class="anchored" data-anchor-id="voxel-based-morphometry-vbm">Voxel-based morphometry (VBM)</h5>
<ul>
<li>MRI technique for measuring brain sizes/volumes</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://openi.nlm.nih.gov/imgs/512/130/2927029/PMC2927029_mp2009146f1.png" class="img-fluid figure-img"></p>
<figcaption>Figure 1 from <span class="citation" data-cites="Pomarol-Clotet2010-tq">Pomarol-Clotet et al. (<a href="#ref-Pomarol-Clotet2010-tq" role="doc-biblioref">2010</a>)</span>. Top panel: (a) voxel-based morphometry (VBM) findings. Regions showing significant volume reduction thresholded at P=0.01 in the schizophrenic patients are shown in orange. Bottom panel: (b) functional magnetic resonance imaging (fMRI) findings. Regions are shown where there were significant differences between patients and controls during performance of the n-back task (2-back vs baseline comparison), thresholded at P=0.01. Blue indicates hypoactivation, that is, areas where controls activated significantly more than the patients. Orange indicates areas where the schizophrenic patients showed failure to deactivate in comparison to controls. The right side of the images represents the left side of the brain.</figcaption>
</figure>
</div>
<ul>
<li>Volume differences in schizophrenics vs.&nbsp;controls</li>
<li>Colored portions are statistical maps placed on top of a base structural map.</li>
<li>Maps (a) provide information about the comparison in brain volumes between patients and controls in those areas, and in (b) functional imaging differences in an n-back task.</li>
</ul>
</section>
</section>
</section>
<section id="mapping-the-wiring-diagram-connectome" class="level3">
<h3 class="anchored" data-anchor-id="mapping-the-wiring-diagram-connectome">Mapping the wiring diagram (“connectome”)</h3>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://cdn.sparkfun.com/r/600-600/assets/1/e/8/c/c/51cdc629ce395f6b4f000000.png" class="lightbox" data-gallery="quarto-lightbox-gallery-19" title="https://learn.sparkfun.com/tutorials/how-to-read-a-schematic/all"><img src="https://cdn.sparkfun.com/r/600-600/assets/1/e/8/c/c/51cdc629ce395f6b4f000000.png" class="img-fluid figure-img" alt="https://learn.sparkfun.com/tutorials/how-to-read-a-schematic/all"></a></p>
<figcaption>https://learn.sparkfun.com/tutorials/how-to-read-a-schematic/all</figcaption>
</figure>
</div>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/jonas-kording-2017.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-20" title="@Jonas2017-sk"><img src="../include/img/jonas-kording-2017.jpg" class="img-fluid figure-img" alt="Jonas &amp; Kording (2017)"></a></p>
<figcaption><span class="citation" data-cites="Jonas2017-sk">Jonas &amp; Kording (<a href="#ref-Jonas2017-sk" role="doc-biblioref">2017</a>)</span></figcaption>
</figure>
</div>
<section id="microscale" class="level4">
<h4 class="anchored" data-anchor-id="microscale">Microscale</h4>
<p>Retrograde (output -&gt; input) vs.&nbsp;anterograde (input -&gt; output) tracers</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/retrograde-anterograde-tracers.png" class="lightbox" data-gallery="quarto-lightbox-gallery-21" title="http://openi.nlm.nih.gov/imgs/512/348/3176268/3176268_1471-2105-12-351-2.png"><img src="../include/img/retrograde-anterograde-tracers.png" class="img-fluid figure-img" alt="http://openi.nlm.nih.gov/imgs/512/348/3176268/3176268_1471-2105-12-351-2.png"></a></p>
<figcaption>http://openi.nlm.nih.gov/imgs/512/348/3176268/3176268_1471-2105-12-351-2.png</figcaption>
</figure>
</div>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/nvXuq9jRWKE" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/embed/nvXuq9jRWKE" class="uri">https://www.youtube.com/embed/nvXuq9jRWKE</a></p>
</section>
<section id="diffusion-tensor-imaging-dti" class="level4">
<h4 class="anchored" data-anchor-id="diffusion-tensor-imaging-dti">Diffusion Tensor Imaging (DTI)</h4>
<ul>
<li>Structural MRI technique</li>
<li>Diffusion tensor: measurement of spatial pattern of <span class="math inline">\(H_2O\)</span> diffusion in small volume</li>
<li>Uniform (“isotropic”) vs.&nbsp;non-uniform (“anisotropic”)</li>
<li>Strong anisotropy suggests large # of axons with similar orientations (fiber tracts)</li>
</ul>
<p><a href="http://1.bp.blogspot.com/-YO3h0cRqTEc/UIRKEC_0N8I/AAAAAAAABQs/dEPzCbcyuCo/s1600/FA_tensor.png" class="lightbox" data-gallery="quarto-lightbox-gallery-22"><img src="http://1.bp.blogspot.com/-YO3h0cRqTEc/UIRKEC_0N8I/AAAAAAAABQs/dEPzCbcyuCo/s1600/FA_tensor.png" class="img-fluid"></a></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-23" title="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg"><img src="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg" class="img-fluid figure-img" alt="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg"></a></p>
<figcaption>https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg</figcaption>
</figure>
</div>
</section>
<section id="visualizing-the-connectome" class="level4">
<h4 class="anchored" data-anchor-id="visualizing-the-connectome">Visualizing the connectome</h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://journals.plos.org/plosone/article/figure/image?size=large&amp;download=&amp;id=10.1371/journal.pone.0135247.g002" class="lightbox" data-gallery="quarto-lightbox-gallery-24" title="Figure 2 from @Bonilha2015-qx. Fig 2. Link-wise ICCs. Each matrix entry represents the ICC observed for the white matter link between the gray matter ROI in the row and the gray matter ROI in the column."><img src="https://journals.plos.org/plosone/article/figure/image?size=large&amp;download=&amp;id=10.1371/journal.pone.0135247.g002" class="img-fluid figure-img" alt="Figure 2 from Bonilha et al. (2015). Fig 2. Link-wise ICCs. Each matrix entry represents the ICC observed for the white matter link between the gray matter ROI in the row and the gray matter ROI in the column."></a></p>
<figcaption>Figure 2 from <span class="citation" data-cites="Bonilha2015-qx">Bonilha et al. (<a href="#ref-Bonilha2015-qx" role="doc-biblioref">2015</a>)</span>. Fig 2. Link-wise ICCs. Each matrix entry represents the ICC observed for the white matter link between the gray matter ROI in the row and the gray matter ROI in the column.</figcaption>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="http://www.humanconnectome.org/storage/app/media/news/2015/09/CCA_mode.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-25" title="http://www.humanconnectome.org"><img src="http://www.humanconnectome.org/storage/app/media/news/2015/09/CCA_mode.jpg" class="img-fluid figure-img" alt="http://www.humanconnectome.org"></a></p>
<figcaption>http://www.humanconnectome.org</figcaption>
</figure>
</div>
</section>
</section>
</section>
<section id="functional-methods" class="level2">
<h2 class="anchored" data-anchor-id="functional-methods">Functional methods</h2>
<ul>
<li>Recording from the brain</li>
<li>Interfering with the brain</li>
<li>Stimulating the brain</li>
<li>Simulating the brain</li>
</ul>
<section id="recording-from-the-brain" class="level3">
<h3 class="anchored" data-anchor-id="recording-from-the-brain">Recording from the brain</h3>
<section id="singlemulti-unit-recording" class="level4">
<h4 class="anchored" data-anchor-id="singlemulti-unit-recording">Single/multi-unit Recording</h4>
<ul>
<li>Microelectrodes + amplification</li>
<li>Small numbers of nerve cells</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://ars.els-cdn.com/content/image/1-s2.0-S2211124721001996-fx1.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-26" title="Figure 1 from @Nishimura2021-ne"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S2211124721001996-fx1.jpg" class="img-fluid figure-img" alt="Figure 1 from Nishimura, Ikegaya, &amp; Sasaki (2021)"></a></p>
<figcaption>Figure 1 from <span class="citation" data-cites="Nishimura2021-ne">Nishimura et al. (<a href="#ref-Nishimura2021-ne" role="doc-biblioref">2021</a>)</span></figcaption>
</figure>
</div>
<ul>
<li>What does neuron X respond to?</li>
<li>How does firing frequency, timing vary with behavior?</li>
<li>Great temporal (ms), spatial resolution (um)</li>
<li>Invasive</li>
<li>Rarely suitable for humans, but…</li>
</ul>
</section>
<section id="electrocorticography-ecog" class="level4">
<h4 class="anchored" data-anchor-id="electrocorticography-ecog"><a href="https://en.wikipedia.org/wiki/Electrocorticography">Electrocorticography (ECoG)</a></h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.researchgate.net/profile/Milena_Korostenskaja/publication/281907202/figure/fig2/AS:613868334223375@1523369024308/Grid-electrodes-A-Craniotomy-performed-for-electrocorticography-ECoG-grid-electrode_Q640.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-27" title="Grid electrodes: (A) Craniotomy performed for electrocorticography (ECoG) grid electrode placement in epilepsy surgery candidate at Comprehensive Epilepsy Program, Florida Hospital for Children, Orlando, Florida, United States. (B) ECoG electrode grids placed directly on the brain surface. They will be used during presurgical monitoring for localizing seizure onset zone. The same electrodes are stimulated during electrical cortical stimulation mapping for identification of eloquent cortex. The ECoG signal recorded from these grids is separated in a different stream and used for real-time functional mapping (RTFM). (C) 3D reconstruction of the brain with overlaid grid electrodes. This reconstruction is used for creating RTFM montage."><img src="https://www.researchgate.net/profile/Milena_Korostenskaja/publication/281907202/figure/fig2/AS:613868334223375@1523369024308/Grid-electrodes-A-Craniotomy-performed-for-electrocorticography-ECoG-grid-electrode_Q640.jpg" class="img-fluid figure-img" alt="Grid electrodes: (A) Craniotomy performed for electrocorticography (ECoG) grid electrode placement in epilepsy surgery candidate at Comprehensive Epilepsy Program, Florida Hospital for Children, Orlando, Florida, United States. (B) ECoG electrode grids placed directly on the brain surface. They will be used during presurgical monitoring for localizing seizure onset zone. The same electrodes are stimulated during electrical cortical stimulation mapping for identification of eloquent cortex. The ECoG signal recorded from these grids is separated in a different stream and used for real-time functional mapping (RTFM). (C) 3D reconstruction of the brain with overlaid grid electrodes. This reconstruction is used for creating RTFM montage."></a></p>
<figcaption>Grid electrodes: (A) Craniotomy performed for electrocorticography (ECoG) grid electrode placement in epilepsy surgery candidate at Comprehensive Epilepsy Program, Florida Hospital for Children, Orlando, Florida, United States. (B) ECoG electrode grids placed directly on the brain surface. They will be used during presurgical monitoring for localizing seizure onset zone. The same electrodes are stimulated during electrical cortical stimulation mapping for identification of eloquent cortex. The ECoG signal recorded from these grids is separated in a different stream and used for real-time functional mapping (RTFM). (C) 3D reconstruction of the brain with overlaid grid electrodes. This reconstruction is used for creating RTFM montage.</figcaption>
</figure>
</div>
<hr>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/u50HPRe3rOY" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/watch?v=u50HPRe3rOY" class="uri">https://www.youtube.com/watch?v=u50HPRe3rOY</a></p>
</section>
<section id="positron-emission-tomography-pet" class="level4">
<h4 class="anchored" data-anchor-id="positron-emission-tomography-pet"><a href="https://en.wikipedia.org/wiki/Positron_emission_tomography">Positron Emission Tomography (PET)</a></h4>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/GHLBcCv4rqk" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/embed/GHLBcCv4rqk" class="uri">https://www.youtube.com/embed/GHLBcCv4rqk</a></p>
<hr>
<ul>
<li>Radioactive tracers (glucose, oxygen)</li>
<li>Positron decay activates paired detectors</li>
<li>Tomographic techniques reconstruct 3D geometry</li>
<li>Experimental condition - control</li>
<li>Average across individuals</li>
<li>Temporal (~ s) and spatial (mm-cm) resolution worse than fMRI</li>
<li>Radioactive exposures + mildly invasive</li>
<li>Dose &lt; airline crew exposure in 1 yr</li>
</ul>
<section id="example-neurotransmitter-receptor-binding-and-behavior" class="level5">
<h5 class="anchored" data-anchor-id="example-neurotransmitter-receptor-binding-and-behavior">Example: Neurotransmitter receptor binding and behavior</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41398-021-01559-5/MediaObjects/41398_2021_1559_Fig3_HTML.png?as=webp" class="lightbox" data-gallery="quarto-lightbox-gallery-28" title="Figure 3 from @Kantonen2021-to. The blue outline marks brain regions where lower [11C]carfentanil binding potential (BPND) associated with higher External eating score, age and PET scanner as nuisance covariates, cluster forming threshold p < 0.01, FWE corrected. In the red–yellow T-score scale shown are also additional bilateral associations significant with more lenient cluster-defining threshold (p < 0.05, FWE corrected) for visualization purposes."><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41398-021-01559-5/MediaObjects/41398_2021_1559_Fig3_HTML.png?as=webp" class="img-fluid figure-img" alt="Figure 3 from Kantonen et al. (2021). The blue outline marks brain regions where lower [11C]carfentanil binding potential (BPND) associated with higher External eating score, age and PET scanner as nuisance covariates, cluster forming threshold p < 0.01, FWE corrected. In the red–yellow T-score scale shown are also additional bilateral associations significant with more lenient cluster-defining threshold (p < 0.05, FWE corrected) for visualization purposes."></a></p>
<figcaption>Figure 3 from <span class="citation" data-cites="Kantonen2021-to">Kantonen et al. (<a href="#ref-Kantonen2021-to" role="doc-biblioref">2021</a>)</span>. The blue outline marks brain regions where lower [11C]carfentanil binding potential (BPND) associated with higher External eating score, age and PET scanner as nuisance covariates, cluster forming threshold p &lt; 0.01, FWE corrected. In the red–yellow T-score scale shown are also additional bilateral associations significant with more lenient cluster-defining threshold (p &lt; 0.05, FWE corrected) for visualization purposes.</figcaption>
</figure>
</div>
<blockquote class="blockquote">
<p>Our main finding was that higher DEBQ scores were associated with lower central availability of MORs and CB1Rs in healthy, nonobese humans. MOR and CB1R systems however showed distinct patterns of associations with specific dimensions of self-reported eating: While CB1Rs were associated in general negatively with different DEBQ subscale scores (and most saliently with the Total DEBQ score), MORs were specifically and negatively associated with externally driven eating only. Our results support the view that variation in endogenous opioid and endocannabinoid systems explain interindividual variation in feeding, with distinct effects on eating behavior measured with DEBQ.</p>
<p><span class="citation" data-cites="Kantonen2021-to">Kantonen et al. (<a href="#ref-Kantonen2021-to" role="doc-biblioref">2021</a>)</span></p>
</blockquote>
</section>
<section id="comparing-pet-and-fmri" class="level5">
<h5 class="anchored" data-anchor-id="comparing-pet-and-fmri">Comparing PET and fMRI</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://ars.els-cdn.com/content/image/1-s2.0-S105381191830586X-gr2_lrg.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-29" title="Figure 2 from @Rischka2018-pk. Fig. 2. Task-specific changes during finger tapping and visual stimulation obtained with fPET and fMRI across all subjects. Good agreement between CMRGlu and BOLD was observed for primary motor and visual cortices. However, in secondary areas (e.g., supplementary motor area, cerebellum, secondary visual areas) significant changes were only detected with fMRI but not with fPET (Table 2). Statistical maps were corrected for multiple comparisons at p<0.05 FWE corrected voxel-level."><img src="https://ars.els-cdn.com/content/image/1-s2.0-S105381191830586X-gr2_lrg.jpg" class="img-fluid figure-img" alt="Figure 2 from Rischka et al. (2018). Fig. 2. Task-specific changes during finger tapping and visual stimulation obtained with fPET and fMRI across all subjects. Good agreement between CMRGlu and BOLD was observed for primary motor and visual cortices. However, in secondary areas (e.g., supplementary motor area, cerebellum, secondary visual areas) significant changes were only detected with fMRI but not with fPET (Table 2). Statistical maps were corrected for multiple comparisons at p<0.05 FWE corrected voxel-level."></a></p>
<figcaption>Figure 2 from <span class="citation" data-cites="Rischka2018-pk">Rischka et al. (<a href="#ref-Rischka2018-pk" role="doc-biblioref">2018</a>)</span>. Fig. 2. Task-specific changes during finger tapping and visual stimulation obtained with fPET and fMRI across all subjects. Good agreement between CMRGlu and BOLD was observed for primary motor and visual cortices. However, in secondary areas (e.g., supplementary motor area, cerebellum, secondary visual areas) significant changes were only detected with fMRI but not with fPET (Table 2). Statistical maps were corrected for multiple comparisons at p&lt;0.05 FWE corrected voxel-level.</figcaption>
</figure>
</div>
<blockquote class="blockquote">
<p>The brain’s energy budget can be non-invasively assessed with different imaging modalities such as functional MRI (fMRI) and PET (fPET), which are sensitive to oxygen and glucose demands, respectively. The introduction of hybrid PET/MRI systems further enables the simultaneous acquisition of these parameters…The absence of a correlation and the different activation pattern between fPET and fMRI suggest that glucose metabolism and oxygen demand capture complementary aspects of energy demands.</p>
<p><span class="citation" data-cites="Rischka2018-pk">Rischka et al. (<a href="#ref-Rischka2018-pk" role="doc-biblioref">2018</a>)</span></p>
</blockquote>
</section>
</section>
<section id="functional-magnetic-resonance-imaging-fmri" class="level4">
<h4 class="anchored" data-anchor-id="functional-magnetic-resonance-imaging-fmri">Functional Magnetic Resonance Imaging (fMRI)</h4>
<ul>
<li>Neural activity -&gt; local <span class="math inline">\(O_2\)</span> consumption increase</li>
<li><em>Blood Oxygen Level Dependent (BOLD)</em> response</li>
<li>Oxygenated vs.&nbsp;deoxygenated hemoglobin ≠ magnetic susceptibility</li>
<li>How do regional blood <span class="math inline">\(O_2\)</span> levels (&amp; flow &amp; volume) vary with behavior X?</li>
<li>MRI “signals” relate to the speed (1/T) of “relaxation” of the perturbed nuclei to their state of alignment with the main (<span class="math inline">\(B_0\)</span>) magnetic field.</li>
<li>Imaging protocols emphasize different time constants of this relaxation (<span class="math inline">\(T1\)</span>, <span class="math inline">\(T2\)</span>, <span class="math inline">\(T2^*\)</span>); <span class="math inline">\(T^{2^*}\)</span> for BOLD imaging</li>
</ul>
<section id="evaluating-fmri" class="level5">
<h5 class="anchored" data-anchor-id="evaluating-fmri">Evaluating fMRI</h5>
<ul>
<li>Non-invasive, but expensive</li>
<li>Moderate but improving (mm) spatial, temporal (~sec) resolution</li>
<li>Spatial limits due to
<ul>
<li>field strength (@ 3T <span class="math inline">\(~3mm^3\)</span> voxel)</li>
<li>Physiology of hemodynamic response</li>
</ul></li>
<li>Temporal limits due to
<ul>
<li>Hemodynamic Response Function (HRF): ~ 1s delay plus 3-6 s ramp-up</li>
<li>Speed of image acquisition</li>
</ul></li>
<li><em>Indirect</em> measure of neural activity</li>
</ul>
</section>
<section id="hemodynamic-response-function-hrf" class="level5">
<h5 class="anchored" data-anchor-id="hemodynamic-response-function-hrf">Hemodynamic Response Function (HRF)</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/hemodynamic-response-function.png" class="lightbox" data-gallery="quarto-lightbox-gallery-30" title="https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png"><img src="../include/img/hemodynamic-response-function.png" class="img-fluid figure-img" alt="https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png"></a></p>
<figcaption>https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png</figcaption>
</figure>
</div>
<hr>
<p><a href="https://mriquestions.com/uploads/3/4/5/7/34572113/convolution-of-3-events_orig.gif" class="lightbox" data-gallery="quarto-lightbox-gallery-31" title="Generate “predicted” BOLD response to event; compare to actual."><img src="https://mriquestions.com/uploads/3/4/5/7/34572113/convolution-of-3-events_orig.gif" class="img-fluid" alt="Generate “predicted” BOLD response to event; compare to actual."></a> Generate “predicted” BOLD response to event; compare to actual</p>
</section>
<section id="higher-field-strengths-3-tesla-vs.-7-tesla" class="level5">
<h5 class="anchored" data-anchor-id="higher-field-strengths-3-tesla-vs.-7-tesla">Higher field strengths (3 Tesla vs.&nbsp;7 Tesla)</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3629563/bin/gr3.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-32" title="Figure from @Sladky2013-bu"><img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3629563/bin/gr3.jpg" class="img-fluid figure-img" alt="Figure from Sladky et al. (2013)"></a></p>
<figcaption>Figure from <span class="citation" data-cites="Sladky2013-bu">Sladky et al. (<a href="#ref-Sladky2013-bu" role="doc-biblioref">2013</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="fmri-studies-often-statistically-underpowered" class="level5">
<h5 class="anchored" data-anchor-id="fmri-studies-often-statistically-underpowered">fMRI studies often statistically underpowered</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://journals.plos.org/plosbiology/article/file?id=10.1371/journal.pbio.2000797.g003&amp;type=large" class="lightbox" data-gallery="quarto-lightbox-gallery-33" title="Figure 3 from @Szucs2017-fc"><img src="https://journals.plos.org/plosbiology/article/file?id=10.1371/journal.pbio.2000797.g003&amp;type=large" class="img-fluid figure-img" alt="Figure 3 from Szucs &amp; Ioannidis (2017)"></a></p>
<figcaption>Figure 3 from <span class="citation" data-cites="Szucs2017-fc">Szucs &amp; Ioannidis (<a href="#ref-Szucs2017-fc" role="doc-biblioref">2017</a>)</span></figcaption>
</figure>
</div>
<blockquote class="blockquote">
<p>Assuming a realistic range of prior probabilities for null hypotheses, false report probability is likely to exceed 50% for the whole literature.</p>
<p><span class="citation" data-cites="Szucs2017-fc">Szucs &amp; Ioannidis (<a href="#ref-Szucs2017-fc" role="doc-biblioref">2017</a>)</span></p>
</blockquote>
<ul>
<li>Solutions
<ul>
<li>Make data, materials (analysis code) more widely and openly available</li>
<li><a href="https://openneuro.org">OpenNeuro.org</a>, <a href="https://www.humanconnectomeproject.org/">Human Connectome Project</a>, <a href="https://databrary.org">Databrary.org</a>, etc.</li>
<li>Reuse shared data (e.g., <a href="https://abcdstudy.org/">Adolescent Brain &amp; Cognitive Development (ABCD) Study</a>)</li>
<li>Increase sample sizes, improve detection of small effects</li>
</ul></li>
</ul>
</section>
</section>
<section id="functional-near-infrared-spectroscopy-fnirs" class="level4">
<h4 class="anchored" data-anchor-id="functional-near-infrared-spectroscopy-fnirs">Functional Near-infrared Spectroscopy (fNIRS)</h4>
<ul>
<li>Near infrared light penetrates scalp and skull, refracted by brain tissue</li>
<li>Returned signal altered by blood <span class="math inline">\(O_2\)</span> levels</li>
<li>Time course (temporal resolution) ~ BOLD fMRI</li>
<li>Spatial resolution low</li>
<li>More suitable than fMRI for pediatric populations (less susceptible to movement artefact)</li>
</ul>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.clinicalguidelines.scot.nhs.uk/media/3702/nirs-fig1.png" class="lightbox" data-gallery="quarto-lightbox-gallery-34" title="https://www.clinicalguidelines.scot.nhs.uk/nhsggc-guidelines/nhsggc-guidelines/intensive-and-critical-care/near-infrared-spectroscopy-nirs-guideline/"><img src="https://www.clinicalguidelines.scot.nhs.uk/media/3702/nirs-fig1.png" class="img-fluid figure-img" alt="https://www.clinicalguidelines.scot.nhs.uk/nhsggc-guidelines/nhsggc-guidelines/intensive-and-critical-care/near-infrared-spectroscopy-nirs-guideline/"></a></p>
<figcaption>https://www.clinicalguidelines.scot.nhs.uk/nhsggc-guidelines/nhsggc-guidelines/intensive-and-critical-care/near-infrared-spectroscopy-nirs-guideline/</figcaption>
</figure>
</div>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://static.wixstatic.com/media/d5c1b3_ef3d7c4006f247f1b455d90ca380da08~mv2.png/v1/fill/w_703,h_427,al_c,lg_1,q_85,enc_auto/Principles%20of%20NIRs_tif.png" class="lightbox" data-gallery="quarto-lightbox-gallery-35" title="https://www.fnndsc.org/principles-of-nirs"><img src="https://static.wixstatic.com/media/d5c1b3_ef3d7c4006f247f1b455d90ca380da08~mv2.png/v1/fill/w_703,h_427,al_c,lg_1,q_85,enc_auto/Principles%20of%20NIRs_tif.png" class="img-fluid figure-img" alt="https://www.fnndsc.org/principles-of-nirs"></a></p>
<figcaption>https://www.fnndsc.org/principles-of-nirs</figcaption>
</figure>
</div>
</section>
<section id="electroencephalography-eeg" class="level4">
<h4 class="anchored" data-anchor-id="electroencephalography-eeg">Electroencephalography (EEG)</h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://upload.wikimedia.org/wikipedia/commons/2/26/Spike-waves.png" class="img-fluid figure-img"></p>
<figcaption>https://en.wikipedia.org/wiki/Electroencephalography</figcaption>
</figure>
</div>
<ul>
<li>How does it work?</li>
<li>Electrodes on scalp or brain surface</li>
</ul>
<section id="what-does-eeg-measure" class="level5">
<h5 class="anchored" data-anchor-id="what-does-eeg-measure">What does EEG measure?</h5>
<ul>
<li>Voltage <em>differences</em> between source and reference electrode</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://naxon.s3.amazonaws.com/uploads/post/image/71/large_International_10-10_system.jpg" class="img-fluid figure-img"></p>
<figcaption>https://naxonlabs.com/blog/understanding-strategic-placement-sensors-eeg-devices</figcaption>
</figure>
</div>
<ul>
<li>Combined activity of huge # of neurons</li>
<li>Current/voltage gradients between <em>apical</em> (near surface) dendrites and <em>basal</em> (deeper) dendrites and cell body/soma</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.biorxiv.org/content/biorxiv/early/2020/07/04/2020.07.01.181875/F1.large.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-36" title="@Naess2020-su"><img src="https://www.biorxiv.org/content/biorxiv/early/2020/07/04/2020.07.01.181875/F1.large.jpg" class="img-fluid figure-img" alt="Næss et al. (2020)"></a></p>
<figcaption><span class="citation" data-cites="Naess2020-su">Næss et al. (<a href="#ref-Naess2020-su" role="doc-biblioref">2020</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="collecting-eeg" class="level5">
<h5 class="anchored" data-anchor-id="collecting-eeg">Collecting EEG</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/baby-eeg.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-37" title="https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg"><img src="../include/img/baby-eeg.jpg" class="img-fluid figure-img" alt="https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg"></a></p>
<figcaption>https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg</figcaption>
</figure>
</div>
</section>
<section id="evaluating-eeg" class="level5">
<h5 class="anchored" data-anchor-id="evaluating-eeg">Evaluating EEG</h5>
<ul>
<li>High temporal, poor spatial resolution</li>
<li>Analyze activity in different ‘bands’ of frequencies (via <a href="https://en.wikipedia.org/wiki/Fourier_transform">Fourier</a> analysis)
<ul>
<li>LOW: deep sleep (delta or <span class="math inline">\(\delta\)</span> band)</li>
<li>MIDDLE: Quiet, alert state (alpha <span class="math inline">\(\alpha\)</span> band)</li>
<li>HIGHER: Sensorimotor activity reflecting observed actions? (mu or <span class="math inline">\(\mu\)</span> band), <span class="citation" data-cites="Hobson2017-oj">(<a href="#ref-Hobson2017-oj" role="doc-biblioref">Hobson &amp; Bishop, 2017</a>)</span></li>
<li>HIGHER STILL: “Binding” information across senses or plasticity? (gamma or <span class="math inline">\(\gamma\)</span> band), <span class="citation" data-cites="Amo2017-tz">(<a href="#ref-Amo2017-tz" role="doc-biblioref">Amo et al., 2017</a>)</span></li>
</ul></li>
</ul>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/eeg-frequency.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-38" title="https://www.peakmind.co.uk/images/frequency.jpg"><img src="../include/img/eeg-frequency.jpg" class="img-fluid figure-img" alt="https://www.peakmind.co.uk/images/frequency.jpg"></a></p>
<figcaption>https://www.peakmind.co.uk/images/frequency.jpg</figcaption>
</figure>
</div>
</section>
<section id="event-related-potentials-erps" class="level5">
<h5 class="anchored" data-anchor-id="event-related-potentials-erps"><a href="https://en.wikipedia.org/wiki/Event-related_potential">Event-related potentials (ERPs)</a></h5>
<ul>
<li>EEGs time-locked to some event</li>
<li>…Averaged over many such events (trials)</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif" class="lightbox" data-gallery="quarto-lightbox-gallery-39" title="https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif"><img src="https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif" class="img-fluid figure-img" alt="https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif"></a></p>
<figcaption>https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif</figcaption>
</figure>
</div>
</section>
<section id="brain-computer-interface-bci" class="level5 smaller">
<h5 class="smaller anchored" data-anchor-id="brain-computer-interface-bci"><a href="https://computer.howstuffworks.com/brain-computer-interface.htm">Brain Computer Interface (BCI)</a></h5>
<ul>
<li>Based on EEG/ERPs</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png" class="lightbox" data-gallery="quarto-lightbox-gallery-40" title="https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png"><img src="https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png" class="img-fluid figure-img" alt="https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png"></a></p>
<figcaption>https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png</figcaption>
</figure>
</div>
</section>
</section>
<section id="magneto-encephalography-meg" class="level4">
<h4 class="anchored" data-anchor-id="magneto-encephalography-meg">Magneto-encephalography (MEG)</h4>
<ul>
<li>Like EEG, but measuring magnetic fields</li>
<li>High temporal resolution</li>
<li>Magnetic fields (<span class="math inline">\(\vec{B}\)</span>) propagate w/o distortion
<ul>
<li>But oriented orthogonal to electric field (<span class="math inline">\(\vec{E}\)</span>)</li>
</ul></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/35/Onde_electromagnetique.svg/220px-Onde_electromagnetique.svg.png" class="img-fluid figure-img"></p>
<figcaption>https://en.wikipedia.org/wiki/Electromagnetic_field</figcaption>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://www.humanconnectome.org/storage/app/media/art/meg-eeg-fig1-draft.png" class="lightbox" data-gallery="quarto-lightbox-gallery-41" title="https://www.humanconnectome.org/study/hcp-young-adult/project-protocol/meg-eeg"><img src="https://www.humanconnectome.org/storage/app/media/art/meg-eeg-fig1-draft.png" class="img-fluid figure-img" alt="https://www.humanconnectome.org/study/hcp-young-adult/project-protocol/meg-eeg"></a></p>
<figcaption><a href="https://www.humanconnectome.org/study/hcp-young-adult/project-protocol/meg-eeg" class="uri">https://www.humanconnectome.org/study/hcp-young-adult/project-protocol/meg-eeg</a></figcaption>
</figure>
</div>
<ul>
<li><p>Strongest signals from sulci <span class="math inline">\(\perp\)</span> to cortical surface</p></li>
<li><p>vs.&nbsp;EEG that favors signals from neurons <span class="math inline">\(\parallel\)</span> to cortical surface</p></li>
<li><p>Requires shielded chamber (to keep out strong magnetic fields)</p></li>
<li><p>++ cost vs.&nbsp;EEG</p></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-42" title="Wikipedia"><img src="https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg" class="img-fluid figure-img" alt="Wikipedia"></a></p>
<figcaption>Wikipedia</figcaption>
</figure>
</div>
<p>New device minimizes problems with motion</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-019-12486-x/MediaObjects/41467_2019_12486_Fig1_HTML.png" class="lightbox" data-gallery="quarto-lightbox-gallery-43" title="Figure 1 from @Hill2019-ik. Figure 1. A paediatric MEG system: a Experimental setup for three participants age 2- (left), 5- (centre) and 24-years (right). OPMs, housed in a modified bike helmet, measured the MEG signal. b Time-frequency spectra from a single (synthesised gradiometer) channel. Changes in neural oscillations are shown; blue indicates a reduction in oscillatory amplitude relative to baseline; yellow indicates an increase. Note reduction in beta (13–30 Hz) and mu (8–13 Hz) amplitude. c The spatial signature of beta modulation during the period of tactile stimulation (0 s< t < 2 s) (blue overlay)"><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-019-12486-x/MediaObjects/41467_2019_12486_Fig1_HTML.png" class="img-fluid figure-img" alt="Figure 1 from Hill et al. (2019). Figure 1. A paediatric MEG system: a Experimental setup for three participants age 2- (left), 5- (centre) and 24-years (right). OPMs, housed in a modified bike helmet, measured the MEG signal. b Time-frequency spectra from a single (synthesised gradiometer) channel. Changes in neural oscillations are shown; blue indicates a reduction in oscillatory amplitude relative to baseline; yellow indicates an increase. Note reduction in beta (13–30 Hz) and mu (8–13 Hz) amplitude. c The spatial signature of beta modulation during the period of tactile stimulation (0 s< t < 2 s) (blue overlay)"></a></p>
<figcaption>Figure 1 from <span class="citation" data-cites="Hill2019-ik">Hill et al. (<a href="#ref-Hill2019-ik" role="doc-biblioref">2019</a>)</span>. Figure 1. A paediatric MEG system: a Experimental setup for three participants age 2- (left), 5- (centre) and 24-years (right). OPMs, housed in a modified bike helmet, measured the MEG signal. b Time-frequency spectra from a single (synthesised gradiometer) channel. Changes in neural oscillations are shown; blue indicates a reduction in oscillatory amplitude relative to baseline; yellow indicates an increase. Note reduction in beta (13–30 Hz) and mu (8–13 Hz) amplitude. c The spatial signature of beta modulation during the period of tactile stimulation (0 s&lt; t &lt; 2 s) (blue overlay)</figcaption>
</figure>
</div>
</section>
<section id="how-do-eegmeg-and-fmri-relate" class="level4">
<h4 class="anchored" data-anchor-id="how-do-eegmeg-and-fmri-relate">How do EEG/MEG and fMRI relate?</h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/logothetis-2001.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-44" title="@Logothetis2001-ul"><img src="../include/img/logothetis-2001.jpg" class="img-fluid figure-img" alt="Logothetis, Pauls, Augath, Trinath, &amp; Oeltermann (2001)"></a></p>
<figcaption><span class="citation" data-cites="Logothetis2001-ul">Logothetis et al. (<a href="#ref-Logothetis2001-ul" role="doc-biblioref">2001</a>)</span></figcaption>
</figure>
</div>
<ul>
<li>BOLD fMRI likely reflects <strong>presynaptic</strong> (afferent) <em>input</em> to area</li>
<li>EEG/MEG likely reflects <strong>postsynaptic</strong> (efferent) <em>response</em> to those inputs</li>
<li><span class="citation" data-cites="Logothetis2001-ul Logothetis2004-mn">(<a href="#ref-Logothetis2001-ul" role="doc-biblioref">Logothetis et al., 2001</a>; <a href="#ref-Logothetis2004-mn" role="doc-biblioref">Logothetis &amp; Wandell, 2004</a>)</span></li>
</ul>
</section>
</section>
<section id="manipulating-the-brain" class="level3">
<h3 class="anchored" data-anchor-id="manipulating-the-brain">Manipulating the brain</h3>
<ul>
<li>Interfering with it</li>
<li>Stimulating it</li>
</ul>
<section id="interfering-with-the-brain" class="level4">
<h4 class="anchored" data-anchor-id="interfering-with-the-brain">Interfering with the brain</h4>
<ul>
<li>Nature’s“experiments”</li>
<li>Stroke, head injury, tumor</li>
<li>Neuropsychology</li>
</ul>
<section id="phineas-gage" class="level5">
<h5 class="anchored" data-anchor-id="phineas-gage">Phineas Gage</h5>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="http://www.doctorsimpossible.com/wp-content/uploads/2017/11/Phineas-Gage-with-skull-picture.png" class="lightbox" data-gallery="quarto-lightbox-gallery-45" title="http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/"><img src="http://www.doctorsimpossible.com/wp-content/uploads/2017/11/Phineas-Gage-with-skull-picture.png" class="img-fluid figure-img" alt="http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/"></a></p>
<figcaption>http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/</figcaption>
</figure>
</div>
<hr>
<p><a href="https://upload.wikimedia.org/wikipedia/en/9/98/The_Man_Who_Mistook_His_Wife_for_a_Hat_cover.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-46"><img src="https://upload.wikimedia.org/wikipedia/en/9/98/The_Man_Who_Mistook_His_Wife_for_a_Hat_cover.jpg" class="img-fluid"></a></p>
</section>
<section id="evaluating-neuropsychological-methods" class="level5">
<h5 class="anchored" data-anchor-id="evaluating-neuropsychological-methods">Evaluating neuropsychological methods</h5>
<ul>
<li>Logic: IF damage to area X impairs performance, THEN region critical for behavior Y</li>
<li><em>Double dissociation</em>: Damage to area Z leaves behavior Y intact</li>
<li>Weak spatial/temporal resolution</li>
</ul>
</section>
</section>
<section id="stimulating-the-brain" class="level4">
<h4 class="anchored" data-anchor-id="stimulating-the-brain">Stimulating the brain</h4>
<ul>
<li>Electrical (<strong>Direct Current Stimulation - DCS</strong>)</li>
<li>Pharmacological</li>
<li>Magnetic (<strong>Transcranial magnetic stimulation - TMS</strong>)</li>
<li>Spatial/temporal resolution?</li>
<li>Assume stimulation mimics natural activity?</li>
</ul>
<section id="deep-brain-stimulation-as-therapy" class="level5">
<h5 class="anchored" data-anchor-id="deep-brain-stimulation-as-therapy">Deep brain stimulation as therapy</h5>
<ul>
<li>Depression</li>
<li>Epilepsy</li>
<li>Parkinson’s Disease</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="../include/img/deep-brain-stimulation.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-47" title="https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg"><img src="../include/img/deep-brain-stimulation.jpg" class="img-fluid figure-img" alt="https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg"></a></p>
<figcaption>https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg</figcaption>
</figure>
</div>
<hr>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/KDjWdtDyz5I" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://youtu.be/KDjWdtDyz5I" class="uri">https://youtu.be/KDjWdtDyz5I</a></p>
</section>
</section>
<section id="optogenetics" class="level4">
<h4 class="anchored" data-anchor-id="optogenetics"><a href="https://en.wikipedia.org/wiki/Optogenetics">Optogenetics</a></h4>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/I64X7vHSHOE" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/embed/I64X7vHSHOE" class="uri">https://www.youtube.com/embed/I64X7vHSHOE</a></p>
<ul>
<li>Gene splicing techniques insert light-sensitive molecules into neuronal membranes</li>
<li>Application of light at specific wavelengths alters neuronal function</li>
<li>Cell-type specific and temporally precise control</li>
<li>Mimics brain activity</li>
</ul>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/FlGbznBmx8M" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://youtu.be/FlGbznBmx8M" class="uri">https://youtu.be/FlGbznBmx8M</a></p>
</section>
</section>
<section id="simulating-the-brain" class="level3">
<h3 class="anchored" data-anchor-id="simulating-the-brain"><em>Sim</em>ulating the brain</h3>
<ul>
<li>Computer/mathematical models of brain function</li>
<li>Example: neural networks</li>
<li>Cheap, noninvasive, can be stimulated or “lesioned”</li>
</ul>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/gn4nRCC9TwQ" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p><a href="https://www.youtube.com/embed/gn4nRCC9TwQ" class="uri">https://www.youtube.com/embed/gn4nRCC9TwQ</a></p>
</section>
<section id="or-synthesizing-one" class="level3">
<h3 class="anchored" data-anchor-id="or-synthesizing-one">Or synthesizing one…</h3>
<section id="human-brain-organoids" class="level4">
<h4 class="anchored" data-anchor-id="human-brain-organoids">Human brain organoids</h4>
<blockquote class="blockquote">
<p>Brain organoids represent a powerful tool for studying human neurological diseases, particularly those that affect brain growth and structure. However, many diseases manifest with clear evidence of physiological and network abnormality in the absence of anatomical changes, raising the question of whether organoids possess sufficient neural network complexity to model these conditions. Here, we explore the network-level functions of brain organoids using calcium sensor imaging and extracellular recording approaches that together reveal the existence of complex network dynamics reminiscent of intact brain preparations. We demonstrate highly abnormal and epileptiform-like activity in organoids derived from induced pluripotent stem cells from individuals with Rett syndrome, accompanied by transcriptomic differences revealed by single-cell analyses. We also rescue key physiological activities with an unconventional neuroregulatory drug, pifithrin-α. Together, these findings provide an essential foundation for the utilization of brain organoids to study intact and disordered human brain network formation and illustrate their utility in therapeutic discovery.</p>
<p><span class="citation" data-cites="Samarasinghe2021-dk">(<a href="#ref-Samarasinghe2021-dk" role="doc-biblioref">Samarasinghe et al., 2021</a>)</span></p>
</blockquote>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00906-5/MediaObjects/41593_2021_906_Fig2_HTML.png?as=webp" class="lightbox" data-gallery="quarto-lightbox-gallery-48" title="Figure 2 from @Samarasinghe2021-dk. a, Schematic illustrating the identification of active neurons by virtue of their Ca2+ transients (I), representation of their network organization (II) and methods used to collect extracellular recordings (III). b, Example of live two-photon microscopy imaging of an H9 hESC-derived fusion organoid demonstrating acquisition of regions of interest (red circles) and the resulting activity profile shown as normalized ΔF/F values, where each line is an individual neuron (middle) and representation of the same data as a colorized amplitude plot (right). c, Addition of 100 μM BMI had a minimal effect on Cx + Cx fusions (top) yet elicited spontaneous synchronization of neural activities in Cx + GE organoids (bottom). d, These synchronizations can be transformed into a normalized amplitude-versus-time plot for quantitative analyses (left) and further visualized as a clustergram following hierarchical clustering of calcium spiking data (right). e, Pooled data of the amplitude measurements. Plots display the full distribution of individual data points. Dashed and dotted lines indicate the median and quartile values, respectively. n = 3 independent experiments for Cx + Cx and Cx + GE. Analysis of variance (ANOVA) P = 0.0011, F = 8.301, d.f. (between columns) = 3 followed by Tukey’s multiple-comparison test; P = 0.0028 for Cx + Cx versus Cx + GE BMI; P = 0.0100 for Cx + Cx BMI versus Cx + GE BMI; **P = 0.0031 for Cx + GE versus Cx + GE BMI. f–h, LFPs measured from a representative Cx + GE fusion revealed robust oscillatory activities at multiple frequencies during a 5-min period, reflected in both raw traces (f) and the spectrogram (g). Spectral density analysis in h demonstrates the presence of multiple distinct oscillatory peaks ranging from ~1–100 Hz. i–k, Cx + Cx fusion organoids by contrast lack measurable oscillatory activities. Representative traces in f–h are taken from three independent experiments and in i–k from four independent experiments. NS, not significant."><img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00906-5/MediaObjects/41593_2021_906_Fig2_HTML.png?as=webp" class="img-fluid figure-img" alt="Figure 2 from Samarasinghe et al. (2021). a, Schematic illustrating the identification of active neurons by virtue of their Ca2+ transients (I), representation of their network organization (II) and methods used to collect extracellular recordings (III). b, Example of live two-photon microscopy imaging of an H9 hESC-derived fusion organoid demonstrating acquisition of regions of interest (red circles) and the resulting activity profile shown as normalized ΔF/F values, where each line is an individual neuron (middle) and representation of the same data as a colorized amplitude plot (right). c, Addition of 100 μM BMI had a minimal effect on Cx + Cx fusions (top) yet elicited spontaneous synchronization of neural activities in Cx + GE organoids (bottom). d, These synchronizations can be transformed into a normalized amplitude-versus-time plot for quantitative analyses (left) and further visualized as a clustergram following hierarchical clustering of calcium spiking data (right). e, Pooled data of the amplitude measurements. Plots display the full distribution of individual data points. Dashed and dotted lines indicate the median and quartile values, respectively. n = 3 independent experiments for Cx + Cx and Cx + GE. Analysis of variance (ANOVA) P = 0.0011, F = 8.301, d.f. (between columns) = 3 followed by Tukey’s multiple-comparison test; P = 0.0028 for Cx + Cx versus Cx + GE BMI; P = 0.0100 for Cx + Cx BMI versus Cx + GE BMI; **P = 0.0031 for Cx + GE versus Cx + GE BMI. f–h, LFPs measured from a representative Cx + GE fusion revealed robust oscillatory activities at multiple frequencies during a 5-min period, reflected in both raw traces (f) and the spectrogram (g). Spectral density analysis in h demonstrates the presence of multiple distinct oscillatory peaks ranging from ~1–100 Hz. i–k, Cx + Cx fusion organoids by contrast lack measurable oscillatory activities. Representative traces in f–h are taken from three independent experiments and in i–k from four independent experiments. NS, not significant."></a></p>
<figcaption>Figure 2 from <span class="citation" data-cites="Samarasinghe2021-dk">Samarasinghe et al. (<a href="#ref-Samarasinghe2021-dk" role="doc-biblioref">2021</a>)</span>. a, Schematic illustrating the identification of active neurons by virtue of their Ca2+ transients (I), representation of their network organization (II) and methods used to collect extracellular recordings (III). b, Example of live two-photon microscopy imaging of an H9 hESC-derived fusion organoid demonstrating acquisition of regions of interest (red circles) and the resulting activity profile shown as normalized ΔF/F values, where each line is an individual neuron (middle) and representation of the same data as a colorized amplitude plot (right). c, Addition of 100 μM BMI had a minimal effect on Cx + Cx fusions (top) yet elicited spontaneous synchronization of neural activities in Cx + GE organoids (bottom). d, These synchronizations can be transformed into a normalized amplitude-versus-time plot for quantitative analyses (left) and further visualized as a clustergram following hierarchical clustering of calcium spiking data (right). e, Pooled data of the amplitude measurements. Plots display the full distribution of individual data points. Dashed and dotted lines indicate the median and quartile values, respectively. n = 3 independent experiments for Cx + Cx and Cx + GE. Analysis of variance (ANOVA) P = 0.0011, F = 8.301, d.f. (between columns) = 3 followed by Tukey’s multiple-comparison test; <strong>P = 0.0028 for Cx + Cx versus Cx + GE BMI; </strong>P = 0.0100 for Cx + Cx BMI versus Cx + GE BMI; **P = 0.0031 for Cx + GE versus Cx + GE BMI. f–h, LFPs measured from a representative Cx + GE fusion revealed robust oscillatory activities at multiple frequencies during a 5-min period, reflected in both raw traces (f) and the spectrogram (g). Spectral density analysis in h demonstrates the presence of multiple distinct oscillatory peaks ranging from ~1–100 Hz. i–k, Cx + Cx fusion organoids by contrast lack measurable oscillatory activities. Representative traces in f–h are taken from three independent experiments and in i–k from four independent experiments. NS, not significant.</figcaption>
</figure>
</div>
</section>
</section>
</section>
<section id="references" class="level2">




</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-Amo2017-tz" class="csl-entry" role="listitem">
Amo, C., De Santiago, L., Zarza Luciáñez, D., León Alonso-Cortés, J. M., Alonso-Alonso, M., Barea, R., &amp; Boquete, L. (2017). Induced gamma band activity from <span>EEG</span> as a possible index of training-related brain plasticity in motor tasks. <em>PloS One</em>, <em>12</em>(10), e0186008. <a href="https://doi.org/10.1371/journal.pone.0186008">https://doi.org/10.1371/journal.pone.0186008</a>
</div>
<div id="ref-Ando2018-ze" class="csl-entry" role="listitem">
Ando, K., Laborde, Q., Brion, J.-P., &amp; Duyckaerts, C. (2018). Chapter 21 - <span>3D</span> imaging in the postmortem human brain with <span>CLARITY</span> and <span>CUBIC</span>. In I. Huitinga &amp; M. J. Webster (Eds.), <em>Handbook of clinical neurology</em> (Vol. 150, pp. 303–317). Elsevier. <a href="https://doi.org/10.1016/B978-0-444-63639-3.00021-9">https://doi.org/10.1016/B978-0-444-63639-3.00021-9</a>
</div>
<div id="ref-Bonilha2015-qx" class="csl-entry" role="listitem">
Bonilha, L., Gleichgerrcht, E., Fridriksson, J., Rorden, C., Breedlove, J. L., Nesland, T., … Focke, N. K. (2015). Reproducibility of the structural brain connectome derived from diffusion tensor imaging. <em>PloS One</em>, <em>10</em>(8), e0135247. <a href="https://doi.org/10.1371/journal.pone.0135247">https://doi.org/10.1371/journal.pone.0135247</a>
</div>
<div id="ref-Hill2019-ik" class="csl-entry" role="listitem">
Hill, R. M., Boto, E., Holmes, N., Hartley, C., Seedat, Z. A., Leggett, J., … Brookes, M. J. (2019). A tool for functional brain imaging with lifespan compliance. <em>Nature Communications</em>, <em>10</em>(1), 4785. <a href="https://doi.org/10.1038/s41467-019-12486-x">https://doi.org/10.1038/s41467-019-12486-x</a>
</div>
<div id="ref-Hobson2017-oj" class="csl-entry" role="listitem">
Hobson, H. M., &amp; Bishop, D. V. M. (2017). The interpretation of mu suppression as an index of mirror neuron activity: Past, present and future. <em>Royal Society Open Science</em>, <em>4</em>(3), 160662. <a href="https://doi.org/10.1098/rsos.160662">https://doi.org/10.1098/rsos.160662</a>
</div>
<div id="ref-Jonas2017-sk" class="csl-entry" role="listitem">
Jonas, E., &amp; Kording, K. P. (2017). Could a neuroscientist understand a microprocessor? <em>PLoS Computational Biology</em>, <em>13</em>(1), e1005268. <a href="https://doi.org/10.1371/journal.pcbi.1005268">https://doi.org/10.1371/journal.pcbi.1005268</a>
</div>
<div id="ref-Kantonen2021-to" class="csl-entry" role="listitem">
Kantonen, T., Karjalainen, T., Pekkarinen, L., Isojärvi, J., Kalliokoski, K., Kaasinen, V., … Nummenmaa, L. (2021). Cerebral <span class="math inline">\(\mu\)</span>-opioid and <span>CB1</span> receptor systems have distinct roles in human feeding behavior. <em>Translational Psychiatry</em>, <em>11</em>(1), 442. <a href="https://doi.org/10.1038/s41398-021-01559-5">https://doi.org/10.1038/s41398-021-01559-5</a>
</div>
<div id="ref-Lee2021-ae" class="csl-entry" role="listitem">
Lee, J. K., Andrews, D. S., Ozonoff, S., Solomon, M., Rogers, S., Amaral, D. G., &amp; Nordahl, C. W. (2021). Longitudinal evaluation of cerebral growth across childhood in boys and girls with autism spectrum disorder. <em>Biological Psychiatry</em>, <em>90</em>(5), 286–294. <a href="https://doi.org/10.1016/j.biopsych.2020.10.014">https://doi.org/10.1016/j.biopsych.2020.10.014</a>
</div>
<div id="ref-lichtman_technicolour_2008" class="csl-entry" role="listitem">
Lichtman, J. W., Livet, J., &amp; Sanes, J. R. (2008). A technicolour approach to the connectome. <em>Nature Reviews Neuroscience</em>, <em>9</em>(6), 417–422. <a href="https://doi.org/10.1038/nrn2391">https://doi.org/10.1038/nrn2391</a>
</div>
<div id="ref-Logothetis2001-ul" class="csl-entry" role="listitem">
Logothetis, N. K., Pauls, J., Augath, M., Trinath, T., &amp; Oeltermann, A. (2001). Neurophysiological investigation of the basis of the <span class="nocase">fMRI</span> signal. <em>Nature</em>, <em>412</em>(6843), 150–157. <a href="https://doi.org/10.1038/35084005">https://doi.org/10.1038/35084005</a>
</div>
<div id="ref-Logothetis2004-mn" class="csl-entry" role="listitem">
Logothetis, N. K., &amp; Wandell, B. A. (2004). Interpreting the <span>BOLD</span> signal. <em>Annu. Rev. Physiol.</em>, <em>66</em>(1), 735–769. <a href="https://doi.org/10.1146/annurev.physiol.66.082602.092845">https://doi.org/10.1146/annurev.physiol.66.082602.092845</a>
</div>
<div id="ref-Naess2020-su" class="csl-entry" role="listitem">
Næss, S., Halnes, G., Hagen, E., Hagler, D. J., Jr, Dale, A. M., Einevoll, G. T., &amp; Ness, T. V. (2020). Biophysical modeling of the neural origin of <span>EEG</span> and <span>MEG</span> signals. <em>bioRxiv</em>, 2020.07.01.181875. <a href="https://doi.org/10.1101/2020.07.01.181875">https://doi.org/10.1101/2020.07.01.181875</a>
</div>
<div id="ref-Nishimura2021-ne" class="csl-entry" role="listitem">
Nishimura, Y., Ikegaya, Y., &amp; Sasaki, T. (2021). Prefrontal synaptic activation during hippocampal memory reactivation. <em>Cell Reports</em>, <em>34</em>(12), 108885. <a href="https://doi.org/10.1016/j.celrep.2021.108885">https://doi.org/10.1016/j.celrep.2021.108885</a>
</div>
<div id="ref-Pomarol-Clotet2010-tq" class="csl-entry" role="listitem">
Pomarol-Clotet, E., Canales-Rodrı́guez, E. J., Salvador, R., Sarró, S., Gomar, J. J., Vila, F., … McKenna, P. J. (2010). Medial prefrontal cortex pathology in schizophrenia as revealed by convergent findings from multimodal imaging. <em>Mol. Psychiatry</em>, <em>15</em>(8), 823–830. <a href="https://doi.org/10.1038/mp.2009.146">https://doi.org/10.1038/mp.2009.146</a>
</div>
<div id="ref-Rischka2018-pk" class="csl-entry" role="listitem">
Rischka, L., Gryglewski, G., Pfaff, S., Vanicek, T., Hienert, M., Klöbl, M., … Hahn, A. (2018). Reduced task durations in functional <span>PET</span> imaging with [<span>18F]FDG</span> approaching that of functional <span>MRI</span>. <em>NeuroImage</em>, <em>181</em>, 323–330. <a href="https://doi.org/10.1016/j.neuroimage.2018.06.079">https://doi.org/10.1016/j.neuroimage.2018.06.079</a>
</div>
<div id="ref-Samarasinghe2021-dk" class="csl-entry" role="listitem">
Samarasinghe, R. A., Miranda, O. A., Buth, J. E., Mitchell, S., Ferando, I., Watanabe, M., … Novitch, B. G. (2021). Identification of neural oscillations and epileptiform changes in human brain organoids. <em>Nature Neuroscience</em>. <a href="https://doi.org/10.1038/s41593-021-00906-5">https://doi.org/10.1038/s41593-021-00906-5</a>
</div>
<div id="ref-Sejnowski2014-aa" class="csl-entry" role="listitem">
Sejnowski, T. J., Churchland, P. S., &amp; Movshon, J. A. (2014). Putting big data to good use in neuroscience. <em>Nat. Neurosci.</em>, <em>17</em>(11), 1440–1441. <a href="https://doi.org/10.1038/nn.3839">https://doi.org/10.1038/nn.3839</a>
</div>
<div id="ref-Sladky2013-bu" class="csl-entry" role="listitem">
Sladky, R., Baldinger, P., Kranz, G. S., Tröstl, J., Höflich, A., Lanzenberger, R., … Windischberger, C. (2013). High-resolution functional <span>MRI</span> of the human amygdala at 7 <span>T</span>. <em>Eur. J. Radiol.</em>, <em>82</em>(5), 728–733. <a href="https://doi.org/10.1016/j.ejrad.2011.09.025">https://doi.org/10.1016/j.ejrad.2011.09.025</a>
</div>
<div id="ref-Susaki2014-xj" class="csl-entry" role="listitem">
Susaki, E. A., Tainaka, K., Perrin, D., Kishino, F., Tawara, T., Watanabe, T. M., … Ueda, H. R. (2014). Whole-brain imaging with single-cell resolution using chemical cocktails and computational analysis. <em>Cell</em>, <em>157</em>(3), 726–739. <a href="https://doi.org/10.1016/j.cell.2014.03.042">https://doi.org/10.1016/j.cell.2014.03.042</a>
</div>
<div id="ref-Szucs2017-fc" class="csl-entry" role="listitem">
Szucs, D., &amp; Ioannidis, J. P. A. (2017). Empirical assessment of published effect sizes and power in the recent cognitive neuroscience and psychology literature. <em>PLoS Biol.</em>, <em>15</em>(3), e2000797. <a href="https://doi.org/10.1371/journal.pbio.2000797">https://doi.org/10.1371/journal.pbio.2000797</a>
</div>
<div id="ref-Zaslavsky2019-ln" class="csl-entry" role="listitem">
Zaslavsky, K., Zhang, W.-B., McCready, F. P., Rodrigues, D. C., Deneault, E., Loo, C., … Ellis, J. (2019). <span>SHANK2</span> mutations associated with autism spectrum disorder cause hyperconnectivity of human neurons. <em>Nature Neuroscience</em>, <em>22</em>(4), 556–564. <a href="https://doi.org/10.1038/s41593-019-0365-8">https://doi.org/10.1038/s41593-019-0365-8</a>
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Copyright © 2025- Rick Gilmore</p>
</div>   
    <div class="nav-footer-center">
<p>Built with <a href="https://quarto.org/">Quarto</a></p>
<div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/psu-psychology/psy-511-scan-fdns-2025-spring/edit/main/src/resources/methods.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/psu-psychology/psy-511-scan-fdns-2025-spring/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/psu-psychology/psy-511-scan-fdns-2025-spring">
      <i class="bi bi-github" role="img">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>